### ISSUE TITLE
The relationship between Preload and Resource Hints should be clarified

### ISSUE THREAD

**Issue by briansmith** [w3c/resource-hints#38]
1. The link to [PRELOAD] at the end of the document is broken.
2. Resource Hints use the `as` from [PRELOAD]. It would be nice to link directly to the definition of the `as` attribute since it is hard to find in [PRELOAD]. In particular, the section describing `as` is not in the TOC for [PRELOAD].
3. It is confusing for somebody new to resource hints and preload to figure out when to use `rel=prefetch` and when to use `rel=preload`. Some clarification of that would be best.
4. IMO, it doesn't make sense to describe `preconnect` in the same document as `prefetch` and `prerender` but have `preload` separate from `preconnect`. I think all four relations ship be described in resource-hints; that is, I think [PRELOAD] should be merged into this document.

**Comment by igrigorik** [comment]
> The link to [PRELOAD] at the end of the document is broken.

That will be fixed as soon as we publish the first draft. /cc @plehegar any blockers here?

> Resource Hints use the as from [PRELOAD].

We're moving away from `as`, see: https://github.com/w3c/preload/pull/26. Once that lands I'll mirror same language in RH.

> It is confusing for somebody new to resource hints and preload to figure out when to use rel=prefetch and when to use rel=preload. Some clarification of that would be best.

Thanks for the feedback, I'll try to make this clearer in the intro of both specs.

> IMO, it doesn't make sense to describe preconnect in the same document as prefetch and prerender but have preload separate from preconnect.

The split was done based on feedback at TPAC. Preload is not a "hint", its MUST execute directive that has different processing implications and policies from other pre\* directives. By contrast, pre{fetch,connect,render} are hints that are processed on best effort basis and with different logic.

**Comment by igrigorik** [comment]
- Preload has been published, and we've defined new 'as' semantics (see latest spec)
- I've reworked the intro in Preload with additional examples.
